draft-ietf-tls-tls13.md

```diff
@@ -2319,8 +2332,8 @@ Servers MUST NOT send a KeyShareEntry for any group not
 indicated in the "supported_groups" extension and
 MUST NOT send a KeyShareEntry when using the "psk_ke" PskKeyExchangeMode.
 If a HelloRetryRequest was received by the client, the client MUST verify that the
-selected NamedGroup matches one that the client sent in its supported_groups
-extension and MUST abort the handshake with an "illegal_parameter" alert otherwise.
+selected NamedGroup in the ServerHello is the same as that in the HelloRetryRequest. If this check
+fails, the client MUST abort the handshake with an "illegal_parameter" alert.
```

Comment:
I'm not sure what's going on here. The old text requires that HRR's group is consistent with CH. The new text is requiring that SH's group is consistent with HRR. Both are true, but why the replacement?